ynput/OpenPype
# AYON: rename Openpype Tools as Custom Tools in Maya Host

## Changelog Description
Rename Openpype Tools as Custom Tools in Maya Host in 
## Additional info
n/a

## Testing notes:
1. build server addon with the latest version of maya addon
2. use the latest maya addon after installing the addon
3. Launch Maya and you would notice that the name of Openpype Tools has renamed as Custom Tools
